## This is the texture description for each scene
## Related pkg: visor
##
## Author: Ferdian Pratama

Header header
int16 count # obj count
textureInfo[] object_texture
int16[] pos_minx
int16[] pos_maxx
int16[] pos_miny
int16[] pos_maxy
sensor_msgs/Image[] image